ecs-agent/tmds/handlers/fault/v1/types/types.go

```diff
@@ -43,6 +45,9 @@ type NetworkBlackholePortRequest struct {
 	Port        *uint16 `json:"Port"`
 	Protocol    *string `json:"Protocol"`
 	TrafficType *string `json:"TrafficType"`
+	// SourcesToFilter is a list including IPv4 addresses or IPv4 CIDR blocks that will be excluded from the
+	// network latency fault.
```

Comment:
nit - `network latency fault.` -> `network blackhole port fault.`

Reply:
Good catch! Let me fix that as a follow-up.